docs/source/model_design/general_reference_material/ensemble_distributions/index.rst

```diff
@@ -229,6 +238,38 @@ modeling <GBD_ensemble_repo_description>` and for :ref:`Vivarium
   - `formatting.py`_ contains helper functions for formatting data and
     converting between data types.
 
+  - Note that as of July 2023 by default, :code:`risk_distributions.py` sets XMIN and XMAX 
+    parameter values to the 0.001 and 0.999 percentile values of a lognormal distribution
+    with the specified mean and standard deviation, `as shown here 
+    <https://github.com/ihmeuw/risk_distributions/blob/242c3c16c403bdf3833a7d3077bd24708150795b/src/risk_distributions/risk_distributions.py#L76>`_
+
+.. note::
+
+  Given that there are two different code bases meant to perform the same task, we have
+  verified that the two code bases indeed accurately replicate one another `in a notebook
+  that can be found here <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/data_prep/risk_distribution/risk%20distribution%20comparison.ipynb>`_. 
+  We found that the two code bases do accurately replicate one another with a few exceptions:
+
+    1. Accurate replication of the mirrored gamma and betasr distributions rely on values 
+    for the XMIN and XMAX parameters. While the GBD R code implementation and Vivarium
+    python code implementation share common default values for the XMIN and XMAX parameters,
+    it is critical to ensure the python implementation reflect custom values used in the
+    GBD implementation to accurately replicate the GBD distributions.
+
+    2. The GBD code for the invweibull distribution relies on an optimization function 
+    that can return unexpected results. The python function appears to solve the 
+    distribution algebraically rather than via optimization and therefore does not 
```

Comment:
The [implementation of get_params in risk_distribution.py](https://github.com/ihmeuw/risk_distributions/blob/242c3c16c403bdf3833a7d3077bd24708150795b/src/risk_distributions/risk_distributions.py#L343) appears to use algebra to compute the mean and variance of an inverse Weibull distribution with given parameters, but it feeds this information into an optimization routine to do the reverse operation, i.e., to find the correct distribution parameters for a given mean and standard deviation.

Reply:
Oh, darn, I see. Maybe it's just that the specific mean/standard deviation I used returned expected results with the python optimization function but not the R optimization function, then. I guess we should run some tests with random mean/sds to see if we get funky behavior with python sometimes too